lib/AST/ASTImporter.cpp

```diff
@@ -7461,8 +7460,10 @@ void ASTImporter::CompleteDecl (Decl *D) {
 }
 
 Decl *ASTImporter::Imported(Decl *From, Decl *To) {
```

Comment:
Is the return value used at all?

Reply:
Of course, it is used all over in the importer, e.g when you import a body of a function then you set up the body on the node by using the returned value.

Reply:
I mean the return value of `Imported` not of `Import`. It is always the same as `To` and not really used.

Reply:
ah okay, I'll try if we can make the return value `void`.